src/components/SearchForm.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { TfiSearch } from "react-icons/tfi";
+import "../styles/SearchForm.css";
+
+function SearchForm({ searchText, setSearchText, onSubmit }) {
+  const handleInputChange = (event) => setSearchText(event.target.value);
+
+  return (
+    <div className="search-form" role="search">
+      <input
+        type="text"
+        onChange={handleInputChange}
+        value={searchText}
+        data-testid="search-input"
+      />
+      <button
+        className="button"
+        type="submit"
```

Comment:
In your app, pressing return on your search form doesn't work. If this is the functionality you wanted, you need to wrap your input and button in a form. Then pressing enter would count as a form submission.
The `type="submit` attribute doesn't do anything as it is and so there's no point in it being here. Again, ask if you need any further explanation